src/App.tsx

```diff
@@ -5,42 +5,29 @@ import usersFromServer from './api/users';
 import todosFromServer from './api/todos';
 import { TodoList } from './components/TodoList';
 import { Todo } from './types';
+import { getUserById, getNewTodoId } from './helpers/helpers';
 
-const getUser = (userId: number) => {
-  const foundedUser = usersFromServer.find(user => (
-    userId === user.id
-  ));
-
-  return foundedUser || null;
-};
-
-const todosWithUser: Todo[] = todosFromServer.map(todo => (
-  {
-    ...todo,
-    user: getUser(todo.userId),
-  }
-));
-
-const newId = (todos: Todo[]) => {
-  const maxId = (todos.map((todo) => (todo.id)));
-
-  return Math.max(...maxId) + 1;
-};
+const todosWithUser: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUserById(todo.userId),
+}));
 
 export const App = () => {
-  const [user, setUser] = useState(0);
+  const [userId, setUserId] = useState(0);
   const [title, setTitle] = useState('');
-  const [todos, setTodos] = useState([...todosWithUser]);
+  const [todos, setTodos] = useState(todosWithUser);
   const [titleError, setTitleError] = useState(false);
   const [userError, setUserError] = useState(false);
 
-  const handleTitleSelect = (event: React.ChangeEvent<HTMLInputElement>) => {
-    setTitle(event.target.value);
-    setTitleError(false);
+  const handleTitleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    if (event.target.value.trim() !== '') {
```

Comment:
Because of this, I can't remove the first symbol in the input.
![image](https://github.com/mate-academy/react_add-todo-form/assets/23013525/ae5b8cee-9cc7-46df-be6d-4321aeab6639)

You should prevent **submitting** wrong data, not **inputting** it.